README.md

```diff
@@ -83,29 +87,27 @@ docker-compose build
 docker-compose up
 ```
 
-laa-crime-means-assessment application will be running on http://localhost:8090
+laa-crime-means-assessment application will be running on http://localhost:8080
 
-### Cloud Platform Set Up
+## Database
 
-It is advisable to have the cloud platform set up locally.
-
-Follow this link to on board yourself with the LAA cloud platform environment. - https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/kubectl-config.html#how-to-use-kubectl-to-connect-to-the-cluster
-
-Configure AWS details using aws cli (command - `aws configure`) Set up AWS Access Key ID & AWS Secret Access Key. All other values can be default.
-
-More detail can be found on https://dsdmoj.atlassian.net/wiki/spaces/LAACP/pages/edit-v2/1756201359.
+This application is run with PostgresSQL using docker compose. PostgresSQL is used solely for static data.
+For database changes, we are using [liquibase]() and all the sql scripts stored in the directory (resources/db/changelog/).
 
-The terraform scripts for the SQS can be found on https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-dev
+All CRUD operations in the MAATDB are run via the [MAAT-API](https://github.com/ministryofjustice/laa-maat-court-data-api)
 
-### Deployment
+## CI/CD
 
 We have configured a CircleCI code pipelines. You can [log in](https://app.circleci.com/pipelines/github/ministryofjustice/laa-crime-means-assessment) from here to access the pipeline.
 
-### Open API
+To make any changes,create a branch and submit the PR. Once the PR is submitted the branch deployment is kicked off under the new branch name.
+On successful build, the image is pushed to AWS ECR and requires approval to deploy to dev.
 
-We have implemented the Open API standard (with Swagger 3). The web link provides a details Rest API with a schema definition. The link can only from local or from dev environment. The swagger link can be found from [here](http://localhost:8090/open-api/docs.html)
+Once the PR is merged with main, the deployment is automatically triggered and requires approval to deploy to desired environment.
```

Comment:
Once the PR is merged with main, the build is automatically deployed to DEV. Further environments deployment require approval.